README.md

```diff
@@ -2,20 +2,30 @@
 
-While `window.print()` is a basic function that triggers the browser's default print dialog, the `navigator.printing` API provides a more comprehensive set of methods and options for querying local printers, submitting print jobs with customization options, managing print jobs, and customizing print workflows. This extended functionality allows developers to have more control over the printing process and enables advanced use cases, such as printing specific pages or sections of a document, setting print options like paper size and orientation, and monitoring the status of print jobs.
+### Customizing Print Settings
+The enhanced features aim to give developers greater influence over the printing process of a document. These capabilities include selecting the printer without requiring user input, printing specific pages or sections of the document, and adjusting print settings such as paper size, orientation, or finishings directly through the script.
 
-### Leveraging Progressive Web Apps
-The new Web API can motivate developers to switch to Progressive Web Apps (PWAs) by offering enhanced printing capabilities that were traditionally limited to native applications (in particular, support for Printer Redirection).
+### Knowing Your Printers
+To enable customization of settings, developers need to have knowledge about the capabilities of the printer. The API provides a convenient method for accessing this information, which enhances the printing experience by allowing developers to retrieve relevant details about what the printer can and cannot do.
+
+### Canceling & Tracking Print Jobs
+When users utilize `window.print()`, they have limited visibility and control over the printing process. However, the API fills this gap by offering the ability to track the progress of print jobs and allowing users to cancel them if needed. This functionality provides users with a way to monitor and influence the state transitions of their print jobs, enhancing their overall printing experience.
+
+### Improving Error Handling
+Similarly to the previous point, the `window.print()` method lacks built-in error handling mechanisms. If a print job fails or encounters an error, the method does not provide meaningful feedback or ways to handle the error effectively.
+
+### Supporting Remote Printing
+The proposed API methods facilitate printer forwarding by enabling the remote client to access essential information about printers on the near side. This allows the remote client to interact with and control those printers as if they were directly connected to the remote client itself, enhancing the seamless integration and operation of printers in the remote environment.
```

Comment:
Explain why using `window.print()` for this is insufficient and creates a poor user experience.

Reply:
Just to make sure -- am I getting it right that we actually don't have a reasonable way of accomplishing this right now without the API? Say, via some obscure video capturing and mouse event redirection :)